NDTensors/ext/NDTensorsMetalExt/adapt.jl

```diff
@@ -7,14 +7,11 @@ end
 # More general than the version in Metal.jl
 function Adapt.adapt_storage(arraytype::Type{<:MtlArray}, xs::AbstractArray)
   # TODO: Do this in one call.
-  arraytype_specified_1 = set_unspecified_parameters(
-    arraytype, Position(1), get_parameter(xs, Position(1))
+  arraytype_specified = specify_parameters(
+    arraytype,
+    get_parameter(xs, Position(1)),
+    get_parameter(xs, Position(2)),
+    get_parameter(xs, Position(3)),
```

Comment:
Can you use `get_parameters(xs)` here?

Reply:
I modified this to use `set_parameters(xs)` and it looks like it works properly. Thanks!